build-bin/dist_to_gh_pages

```diff
@@ -15,26 +15,37 @@
 
 set -ue
 
+ORIGIN_BRANCH='master'
```

Comment:
these variables add confusion. I guess you are extracting them for tests. Please use these conventions if we must override for tests and add a comment that we are doing this.


```
# we never change these during deployment, these variables are only here for ad-hoc tests.
DEFAULT_BRANCH=master
GH_PAGES_BRANCH=gh-pages
```